mars-sim-core/src/main/resources/messages.properties

```diff
@@ -914,6 +912,18 @@ NaturalAttributeType.teaching           = Teaching
 
 NavButtonDisplay.log.mediaTrackerError = NavButtonDisplay Media Tracker Error {0}
 
+LandmarkType.AO = Artificial Object
```

Comment:
It'll be great that one day we offer players the option to turn on and off each of these landmark types, wouldn't it ?

Reply:
Yes I agree. I want it that each layer can support filters. I'm working on ideas for it